src/plugins/plot/MctPlot.vue

```diff
@@ -946,23 +950,34 @@ export default {
             }
         },
 
-        onTickWidthChange(data, fromDifferentObject) {
+        onYTickWidthChange(data, fromDifferentObject) {
             const {width, yAxisId} = data;
             if (yAxisId) {
                 const index = this.yAxes.findIndex(yAxis => yAxis.id === yAxisId);
                 if (fromDifferentObject) {
                 // Always accept tick width if it comes from a different object.
                     this.yAxes[index].tickWidth = width;
                 } else {
+                //Commenting out below cuz If we never make the ticks smaller, then we can't adjust ticks for logMode
                 // Otherwise, only accept tick with if it's larger.
-                    const newWidth = Math.max(width, this.yAxes[index].tickWidth);
```

Comment:
This is correct for when you switch contexts, you'll want to take the new width regardless. This one check is important though so that the plot doesn't jump around when precision on incoming telemetry changes. You'll have to figure out how to specify that context switch.

Looking now at what `fromDifferentObject` is...

Reply:
It looks like `fromDifferentObject` could achieve what you want, but take a look at `updateTicksForceRegeneration` in `MCTTicks.vue`. on a key change, it will force the ticks to regenerate. I'm not sure what the log event is that you would listen to.

Reply:
@davetsay I've put that code back. I think we're good.